lib/src/core/service_helper.dart

```diff
@@ -301,16 +301,16 @@ class ServiceHelper implements Service {
       } else if (value is List?) {
         return MapEntry(
           key,
-          value?.toSet().join(','),
+          value?.map((e) => e.toString()).toList()
         );
       } else if (value is Serializable) {
         return MapEntry(
           key,
-          value.value,
+          value.value.toString(),
```

Comment:
@Kraigo ,

Since the value of an enum that implements Serializable is always a String, isn't toString unnecessary?

Reply:
@Kraigo ,

Sorry, I completely forgot to submit this :) Please check and fix if needed :)

Reply:
@myConsciousness You are right, the Seriazible already has String type for value, and we can't override type inherited classes.
I did this to ensure previous logic and apply toString for all values except list.
But I think toString for string value should return the same string, but this way the code looks cleaner.
Good catch, thanks for review

Reply:
@Kraigo ,

Thanks for checking! And it looks great, I'm going to merge it :)